cli/pkg/cicd/initiator/initiator.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package initiator
+
+import "context"
+
+type Initiator interface {
+	InitResults(ctx context.Context) (string, string, error)
```

Comment:
better to add comment on the purpose of the method and what it returns.

Reply:
I've added some comments.